test/documentation/seqan3_doxygen_cfg.in

```diff
@@ -33,6 +33,7 @@ HTML_TIMESTAMP         = YES
 EXT_LINKS_IN_WINDOW    = YES
 RECURSIVE              = YES
 DISABLE_INDEX          = YES
+DISTRIBUTE_GROUP_DOC   = YES
```

Comment:
Allows to use `@{` and `}@` to copy/paste a documentation to all members of the group.

E.g.

```cpp
    /*!\name Construct from arguments.
     * \{
     */
    //!@{ Constructs from arguments.
    template <class... UTypes>
        requires (sizeof...(Types) == sizeof...(UTypes)) && (std::is_constructible_v<Types, UTypes &> && ...)
    constexpr common_tuple(UTypes & ... other) : tuple_t(other...)
    {}

    template <class... UTypes>
        requires (sizeof...(Types) == sizeof...(UTypes)) && (std::is_constructible_v<Types, UTypes const> && ...)
    constexpr common_tuple(UTypes const & ... other) : tuple_t(other...)
    {}

    template <class... UTypes>
        requires (sizeof...(Types) == sizeof...(UTypes)) && (std::is_constructible_v<Types, UTypes> && ...)
    constexpr common_tuple(UTypes && ... other) : tuple_t(std::forward<UTypes>(other)...)
    {}

    template <class... UTypes>
        requires (sizeof...(Types) == sizeof...(UTypes)) && (std::is_constructible_v<Types, UTypes const> && ...)
    constexpr common_tuple(UTypes const && ... other) : tuple_t(std::forward<UTypes const>(other)...)
    {}
    //!@}
    //!\}
```

All these will have `Constructs from arguments.` as brief.